### PR TITLE
Release notes week of 6/20/2016

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3047](https://openedx.atlassian.net/browse/DOC-3047)

Add a description of your changes with links to any relevant material.
### Date Needed (optional)

6/21/16
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer gives :+1:. 
- [x] Subject matter expert: @bjacobel (items 3 and 4 in lms_0620_2016.rst)
- [x] Doc team review (sanity check/copy edit/dev edit): @lamagnifica 
- [x] Product review: @cptvitamin (item 1 in lms_0620_2016.rst)
- [x] Product review: @sstack22 (item 2 in lms_0620_2016.rst)
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2016/lms/lms_0620_2016.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+* The edX video player now renders all transcript files as plain text. HTML
+  markup, including character entities such as ``&nbsp;`` and styling such as
+  bold (strong) or italic (emphasis), is now removed from each transcript .srt
+  file before the file plays for a video component.
+
+  In addition, the contrast used to display the selected item in the video
```
Comment (cptvitamin) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: @srpearce the change did not actually affect the contrast but how the current state is indicated, so this statement is not true.  When High Contrast mode is enabled on Windows, things like background colors are not displayed as the OS/Browser overrides the CSS provided by the author.  So even though this appears to be related to a change in contrast, its not.  I would say something like "In addition, the method used to indicate the selected item in the video player's menus for speed and transcript language has been changed. This change is intended to cleary indicate the current state of these controls when high contrast mode is enabled on a user's operating system. (:jira:`AC-454`)"

Comment (lamagnifica) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: my bad, I misunderstood @clrux 's description. +1 to Mark's

File: en_us/release_notes/source/2016/lms/lms_0620_2016.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  player's menus for speed and transcript language has been increased. This
+  change is intended to improve the experience of using the video player when
+  high contrast mode is enabled on a user's operating system. (:jira:`AC-454`)
+
+* On the **Account Settings** page, learners can now specify their local time
```
Comment (lamagnifica) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: per @sstack22 this new setting is behind a feature flag -- the conversion is still being built, and outward advance notice hasn't been given yet (yay team!) so please remove this bullet for this week.

File: en_us/release_notes/source/2016/lms/lms_0620_2016.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  zone. All course dates and times will then appear in a learner's local time.
+  If learners do not specify a time zone, course dates and times continue to
+  appear in Coordinated Universal Time (UTC). (:jira:`TNL-4594`)
+
+* In course discussions, large images could sometimes exceed the width of the
+  discussion screen, so that only part of the image was visible. This problem
+  has been resolved. (:jira:`TNL-4778`)
+
+* In course discussions, learners can no longer vote for or report their own
+  posts, responses, and comments. (:jira:`TNL-4703`)
```
Comment (lamagnifica) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: it's kind of a nit, but you can't vote for comments across the board. You can only ever vote for posts and responses.

**Comment by lamagnifica** [comment]
On nit, one thing to delete. 👍 otherwise!

File: en_us/release_notes/source/2016/lms/lms_0620_2016.rst (hunk 1 above)
Comment (srpearce) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: @cptvitamin Would it be correct to say this?

In addition, the indication for selected items in the video player's speed
  and transcript language menus has changed to more clearly show the current
  state of these controls when a user's operating system is in high contrast
  mode.

Or maybe:

In addition, the method for indicating selected items in the video player's speed
  and transcript language menus has changed to more clearly show the current
  state of these controls when a user's operating system is in high contrast
  mode.

File: en_us/release_notes/source/2016/lms/lms_0620_2016.rst [hunk 4]
```diff
@@ -0,0 +1,18 @@
+* The edX video player now renders all transcript files as plain text. HTML
+  markup, including character entities such as ``&nbsp;`` and styling such as
+  bold (strong) or italic (emphasis), is now removed from each transcript .srt
+  file before the transcript plays for a video component.
+
+  In addition, the indication for selected items in the video player's speed
+  and transcript language menus has changed to more clearly show the current
+  state of these controls when a user's operating system is in high contrast
+  mode. (:jira:`AC-454`)
```
Comment (srpearce) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: @cptvitamin Is the above paragraph correct?

Comment (srpearce) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: @cptvitamin Or, I could possibly shorten it even more: In addition, the indication for the selected speed and transcript language is now more clear when a user's operating system is in high contrast mode.

Comment (cptvitamin) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: @srpearce yes!  your more concise suggestion works great.

Comment (srpearce) [inline en_us/release_notes/source/2016/lms/lms_0620_2016.rst]: @cptvitamin Great - thanks!

**Pull request merged**